mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MailboxMapperTest.java

```diff
@@ -82,11 +82,28 @@ void setUp() {
     }
 
     @Test
-    void findMailboxByPathWhenAbsentShouldFail() throws MailboxException {
+    void findMailboxByPathWhenAbsentShouldFail() {
         assertThatThrownBy(() -> mailboxMapper.findMailboxByPath(MailboxPath.forUser(BENWA, "INBOX")))
             .isInstanceOf(MailboxNotFoundException.class);
     }
 
+    @Test
+    void createShouldPersistTheMailbox() throws MailboxException {
+        mailboxMapper.create(benwaInboxMailbox);
+        MailboxAssert.assertThat(mailboxMapper.findMailboxByPath(benwaInboxPath)).isEqualTo(benwaInboxMailbox);
+    }
+
+    @Test
+    void createShouldThrowWhenMailboxAlreadyExist() throws MailboxException {
```

Comment:
I don't get why we would want a `create` method to throw. Could you elaborate about the benefits you expect?

Reply:
Actually you are right, in this case we can just drop it silently, no need to throw an exception for this

Reply:
Actually it's a bit more complex than that... I think for now it should still throw an exception, as if not you need to refactor the behavior of the `StoreMailboxManager`, that will dispatch an event in this case as it created a mailbox, even if it didn't, if we don't throw an exception anymore. That starts sounding complex and out of scope of this PR, so I will keep expecting `create` to throw, if you don't mind